packages/mini-browser/src/node/mini-browser-endpoint.ts

```diff
@@ -86,23 +88,30 @@ export class MiniBrowserEndpoint implements BackendApplicationContribution, Mini
     protected readonly handlers: Map<string, MiniBrowserEndpointHandler> = new Map();
 
     configure(app: Application): void {
-        app.get(`${MiniBrowserEndpoint.HANDLE_PATH}*`, async (request, response) => this.response(await this.getUri(request), response));
```

Comment:
leftover: `static HANDLE_PATH = '/mini-browser/';`

Reply:
Should I mark it as deprecated or just remove it? It's public API hence why I left it.

Reply:
I see. I believe deprecating it is a good idea.
Thanks!